airbyte-lib/airbyte_lib/types.py

```diff
@@ -22,8 +22,8 @@
     "time_without_timezone": sqlalchemy.types.TIME,
     # Technically 'object' and 'array' as JSON Schema types, not airbyte types.
     # We include them here for completeness.
-    "object": sqlalchemy.types.VARCHAR,
```

Comment:
just for me to understand, why don't we want the varchar anymore?

Reply:
It's actually not being removed, but the diff is displaying strangely. It should render as "object" type having its mapping changed from `sqlalchemy.types.VARCHAR` to `sqlalchemy.types.JSON`